nix/cardano/alerts.nix

```diff
@@ -253,4 +253,90 @@ in
+        expr = ''sort(node_filesystem_free_bytes{device!="ramfs"} < node_filesystem_size_bytes{device!="ramfs"} * 0.1) / 1024^3'';
+        for = "5m";
+        labels.severity = "page";
+        annotations = {
+          summary = "{{$labels.alias}}: Filesystem is running out of space soon.";
+          description = "{{$labels.alias}} device {{$labels.device}} on {{$labels.mountpoint}} got less than 10% space left on its filesystem.";
+        };
+      }
+      {
+        alert = "node_filesystem_full_in_4h";
+        expr = ''predict_linear(node_filesystem_free_bytes{device!~"ramfs|tmpfs|none",fstype!~"autofs|ramfs|cd9660"}[4h], 4*3600) <= 0'';
+        for = "5m";
+        labels.severity = "page";
+        annotations = {
+          summary = "{{$labels.alias}}: Filesystem is running out of space in 4 hours.";
+          description = "{{$labels.alias}} device {{$labels.device}} on {{$labels.mountpoint}} is running out of space of in approx. 4 hours";
+        };
+      }
+      {
+        alert = "node_filedescriptors_full_in_3h";
+        expr = ''predict_linear(node_filefd_allocated[1h], 3*3600) >= node_filefd_maximum'';
+        for = "20m";
+        labels.severity = "page";
+        annotations = {
+          summary = "{{$labels.alias}} is running out of available file descriptors in 3 hours.";
+          description = "{{$labels.alias}} is running out of available file descriptors in approx. 3 hours";
+        };
+      }
+      {
+        alert = "node_time_unsync";
+        expr = ''abs(node_timex_estimated_error_seconds) > 0.500'';
+        for = "5m";
+        labels.severity = "page";
+        annotations = {
+          summary = "{{$labels.alias}}: Clock out of sync with NTP";
+          description = "{{$labels.alias}} Local clock offset is too large or out of sync with NTP";
+        };
+      }
+      {
+        alert = "http_high_internal_error_rate";
+        expr = ''rate(nginx_vts_server_requests_total{code="5xx"}[5m]) * 50 > on(alias, host) rate(nginx_vts_server_requests_total{code="2xx"}[5m])'';
+        for = "15m";
+        labels.severity = "page";
+        annotations = {
+          summary = "{{$labels.alias}}: High http internal error (code 5xx) rate";
+          description = "{{$labels.alias}}  number of correctly served requests is less than 50 times the number of requests aborted due to an internal server error";
+        };
+      }
+      {
+        alert = "varnish cache too small or ttl too long";
+        expr = ''rate(varnish_main_n_lru_nuked[30m]) > 5 * rate(varnish_main_n_expired[30m])'';
+        for = "1h";
+        labels.severity = "page";
+        annotations = {
+          summary = "{{$labels.alias}}: Too many objects (5 times the number of expiring objects) are being forcefully evicted from varnish cache due to memory constraints.";
+          description = "{{$labels.alias}}: Consider increasing varnish malloc limit or decreasing beresp.ttl";
+        };
+      }
```

Comment:
we probably should have something for db-sync height compared to node tip

Reply:
Yup we're covered there with the db-sync alert attr set above.